src/backend/distributed/metadata/node_metadata.c

```diff
@@ -1330,6 +1331,32 @@ citus_update_node(PG_FUNCTION_ARGS)
 }
 
 
+Datum
+citus_pause_node(PG_FUNCTION_ARGS)
+{
+	CheckCitusVersion(ERROR);
+
+	int32 nodeId = PG_GETARG_INT32(0);
+	List *placementList = NIL;
+
+	WorkerNode *workerNode = FindNodeAnyClusterByNodeId(nodeId);
+	if (workerNode == NULL)
+	{
+		ereport(ERROR, (errcode(ERRCODE_NO_DATA_FOUND),
```

Comment:
I have tests to cover this line and I can see that tests are being executed from the output. But still seeing below warning. What could be the reason?

Tests file result: https://github.com/citusdata/citus/blob/ed40dfe1a7482679aa4bc2068db19cba3655f337/src/test/regress/expected/isolation_citus_pause_node.out#L80C1-L111C18